bingo/bingo-elastic/python/.pylintrc

```diff
@@ -4,6 +4,7 @@ disable=
     C0116, # missing-function-docstring
     C0115, # missing-class-docstring
     R0903, # too-few-public-methods
+    R0913, # too-many-arguments
```

Comment:
We should not add this exclusion. Probably it's better to refactor the code that raises this warning.

Reply:
Thanks, found a way to refactor and escape this warning!